libs/ledger/include/ledger/chaincode/wallet_record.hpp

```diff
@@ -19,15 +19,16 @@
 
 #include "core/byte_array/decoders.hpp"
 #include "ledger/chaincode/deed.hpp"
+#include "logging/logging.hpp"
 
 #include <cstdint>
 #include <map>
 #include <memory>
 
 // TODO(HUT): doesn't putting this here pollute the namespace?
```

Comment:
Could we either solve this or add an issue number?

Reply:
Fixed